Makefile

```diff
@@ -16,9 +16,10 @@ PKG := github.com/vmware-tanzu/velero-plugin-for-aws
 BIN := velero-plugin-for-aws
 
 REGISTRY 	?= velero
-IMAGE 		?= $(REGISTRY)/velero-plugin-for-aws
 VERSION 	?= master
 
+CONTAINER_PLATFORMS ?= amd64 arm arm64 # ppc64le
```

Comment:
should `ppv64le` be uncommented?

Reply:
I think ppc64le needs to be removed entirely; it's not supported by Velero itself.